test/postman.test.ts

```diff
@@ -0,0 +1,120 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { assert } from 'chai';
+import { BigNumber, Contract, ContractFactory } from 'ethers';
+import { starknet, network, ethers } from 'hardhat';
+import {
+  StarknetContractFactory,
+  StarknetContract,
+  HttpNetworkConfig,
+} from 'hardhat/types';
+
+describe('Postman', () => {
+  const user = 1;
+  const networkUrl: string = (network.config as HttpNetworkConfig).url;
+  let L2contractFactory: StarknetContractFactory;
+  let l2contract: StarknetContract;
+  let L1L2Example: ContractFactory;
+  let MockStarknetMessaging: ContractFactory;
+  let mockStarknetMessaging: Contract;
+  let l1l2Example: Contract;
+  let signer: SignerWithAddress;
+
+  before(async function () {
+    L2contractFactory = await starknet.getContractFactory('l1l2');
+    l2contract = await L2contractFactory.deploy();
+
+    const signers = await ethers.getSigners();
+    signer = signers[0];
+
+    MockStarknetMessaging = await ethers.getContractFactory(
+      'MockStarknetMessaging',
+      signer,
+    );
+    mockStarknetMessaging = await MockStarknetMessaging.deploy();
+    await mockStarknetMessaging.deployed();
+
+    L1L2Example = await ethers.getContractFactory('L1L2Example', signer);
+    l1l2Example = await L1L2Example.deploy(mockStarknetMessaging.address);
+    await l1l2Example.deployed();
+  });
+
+  it('should deploy the messaging contract', async () => {
+    const {
+      address: deployedTo,
+      l1_provider: L1Provider,
+    } = await starknet.devnet.loadL1MessagingContract(networkUrl);
+
+    assert.exists(deployedTo);
+    assert(L1Provider === networkUrl, `L1 provider is ${L1Provider}, should be ${networkUrl}`);
```

Comment:
Purely stylistic nit-pick: Why not use `equal`? Error messages would be simpler.